scripts/examples/esp_example.sh

```diff
@@ -31,16 +32,21 @@ fi
 
 source "scripts/activate.sh"
 # shellcheck source=/dev/null
-source "$root"/idf.sh
+cd "$IDF_PATH"
+. ./export.sh
 
+cd "$work_dir"
```

Comment:
Can we use `cd -` instead of using a new variable?

Reply:
Done.